### PR TITLE
Get unread count for user

### PR DESCRIPTION
**Pull request by danischreiber**

### CODE REVIEW

File: lib/rocket_chat/messages/user.rb [hunk 1]
```diff
@@ -126,6 +126,21 @@ def info(user_id: nil, username: nil)
...
       #
+      # users.unread_count REST API
+      # @param [String] user_id Rocket.Chat user id
+      # @param [String] username Username
+      # @return [User]
+      # @raise [HTTPError, StatusError]
+      #
+      def unread_count(user_id: nil, username: nil)
+        response = session.request_json(
+            '/api/v1/users.unreadCount',
```
Comment (abrom) [inline lib/rocket_chat/messages/user.rb]: the build is wanting these two lines to be only indented once vs twice

**Comment by danischreiber** [comment]
Thanks, I resubmitted the PR in rocketchat. We can wait to merge until it
gets in.

**Comment by abrom** [comment]
Hey @danischreiber, it looks like your PR in the rocketchat repo for this has gone stale?? Might be worth pinging them to get a review underway

**Comment by abrom** [comment]
Hey @danischreiber, still no movement on your PR in the rocketchat repo? I can't see the end-point in the users API.

**Pull request closed without merging**